_includes/integrations/react-tech-ref.md

```diff
@@ -52,12 +52,12 @@ The editor accepts the following properties:
 />
 ```
 
-None of the configuration properties are **required** for `tinymce-react` to work. Specify a Tiny Cloud API key using `apiKey` to remove the `This domain is not registered...` warning message.
+None of the configuration properties are **required** for `tinymce-react` to work. Specify a {{site.cloudname}} API key using `apiKey` to remove the `This domain is not registered...` warning message.
 
 #### `apiKey`
-Tiny API key. Required for deployments using the Tiny Cloud to provide the TinyMCE editor.
+{{site.cloudname}} API key. Required for deployments using the {{site.cloudname}} to provide the {{site.productname}} editor.
```

Comment:
If the other variable is correct, than this is inconsistent and probably needs updating (or vice versa).

Reply:
fair, reviewing instances of `accountpage`